geowebcache/sqlite/src/main/java/org/geowebcache/sqlite/Utils.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * <p/>
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Nuno Oliveira, GeoSolutions S.A.S., Copyright 2016
+ */
+package org.geowebcache.sqlite;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.geowebcache.io.ByteArrayResource;
+import org.geowebcache.io.Resource;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * Some utilities objects and functions used internally.
+ */
+final class Utils {
+
+    private static Log LOGGER = LogFactory.getLog(Utils.class);
+
+    private static final ThreadLocal<String> LOG_ID = new ThreadLocal<>();
```

Comment:
I see nothing cleaning this thread local at the end of the request, this might cause cross request pollution and will make it harder to cleanly un-deploy the application.